picamera2/picamera2.py

```diff
@@ -112,41 +112,8 @@ def handle_request(self, flushid=None):
 class Picamera2:
     """Welcome to the PiCamera2 class."""
 
-    DEBUG = logging.DEBUG
-    INFO = logging.INFO
-    WARNING = logging.WARNING
-    ERROR = logging.ERROR
-    CRITICAL = logging.CRITICAL
```

Comment:
Seriously, I am suprised we didn't just glue `sys` and `os` onto this god object while we were at it.